create sprint-24-summary.md (#1068)
raft-tech/TANF-app: docs/Sprint-Review/sprint-24-summary.md

# Sprint 24 Summary
**06/22/2021- 07/06/2021**

### Updates
 - For the week of 6/22 was impacted by Raft team having an onsite all-hands. This resulted in less productivity due to folks being out for travel and the all hands meetings, effectively rendering our sprint a little over 50% productivity.
- NextGen XMS decisions are still blocked by having information from the NextGen Team. We requested documentation by 7/12 and are planning to deprioritizing this work until it is unblocked, continuing to use login.gov in the meantime.

> - The team is able to assess and move foreward on planning for ACF/AMS as of end of this sprint.

Reviewer: ```suggestion
- The team is able to assess and move forward on planning for ACF AMS as of end of this sprint.
```

# Sprint 24 Summary
**06/22/2021- 07/06/2021**

### Updates
 - For the week of 6/22 was impacted by Raft team having an onsite all-hands. This resulted in less productivity due to folks being out for travel and the all hands meetings, effectively rendering our sprint a little over 50% productivity.

> - NextGen XMS decisions are still blocked by having information from the NextGen Team. We have discussed moving forward with Login.gov for the time being and deprioritizing the migration efforts since we continue to be blocked.

Reviewer 2: ```suggestion
- NextGen XMS decisions are still blocked by having information from the NextGen Team. We requested documentation by 7/12 and are planning to deprioritizing this work until it is unblocked, continuing to use login.gov in the meantime.
```

> - The team is able to assess and move foreward on planning for ACF/AMS as of end of this sprint.

Reviewer: ```suggestion
- The team is able to assess and move forward on planning for ACF AMS as of end of this sprint.
```

> - The team is able to assess and move foreward on planning for ACF/AMS as of end of this sprint.

Reviewer 2: I'm not 100% sure this is true. I think it's more like we're still assessing

## Sprint Goals

Continue work from Sprint 23...
Finish up ATO Functionality

- Finish up file download & storage (#416, #818, #833, #834)

> - Wrap up OWASP Scan tickets (#879, #865)

Reviewer: ```suggestion
- Wrap up OWASP Scan tickets (#879, #865, #907,  #1032)
```

- Make decisions on upcoming releases

> - Cron Job options for release 2 (#1011)

Reviewer: ```suggestion
- File transfer options for release 2 (#1011)
```

- NextGen XMS direction for release 1 (Epic #902, #638)

> - Document feasibility and high level estimated tasks that would be needed for ACF/AMS + Next Gen (Blocked) implementation.

Reviewer: ```suggestion
- Document feasibility and high level estimated tasks that would be needed for (1) ACF AMS or (2) ACF AMS/Next Gen (Blocked) implementation.
```

...Then
Attend to Permissions for OFA Admins epic

and then....
Work on DevOPs work for Production readiness

## Merged/Completed (Done/Demo, Closed)
- Dependabot Mass Merge / Improvements [#1023](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1023)

> - As a dev/Tech Lead, I want to have automated deploys to production [#724](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/724)

Reviewer: ```suggestion
```

this ticket was closed because it was a dup to #895 which is still open.

- Set up new dev environments: sandbox, raft-review, qasp-review, a11y [#848](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/848)

## Submitted (QASP Review, OCIO Review)
-[Frontend] Add a download button to the Data Files view [#416](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/416)

> - [Backend] Add endpoint to download a file from S3[#833](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/833)

Reviewer: recommend moving into the done/demo section, since this was approved prior to demo 🚀

Author: done

- As a dev, I want an automated, documented CI process to provision Cloud.gov brokered services[#609](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/609)
- Issue 416: [Frontend] Add a download button to the Data Files view[#859](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/859)

## Moving to Next Sprint (Raft Review, In Progress, Current Sprint Backlog)
**Raft Review**
- Django Admin a11y Fixes (Sprint 1) [#973](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/973)

**Blocked**
- As a dev, I need to know which authentication service we're using (login.gov vs. NextGen XMS)[#638](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/638)
- SPIKE: File transfer options for Tribal MVP [#1011](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1011)

**In Progress**

> - Round 7 (Sprint 24-26) UX Research and Design [#1017](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1017)

Reviewer 2: ```suggestion
- [EPIC] Round 7 (Sprint 24-26) UX Research and Design [#1017](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1017)
```

- As an OFA Admin, I want an accessible, 508-compliant user interface for managing permissions [#892](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/892)
- (UX Sprint 24) Current State Analysis of Error Communication and Regional Staff Workflow Validation [#1018](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1018)
- SPIKE: Authentication Feasibility Research [#1046](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1046)
- Verify Admin Permissions Hierarchy and Roles[#1058](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1058)
- [Devops] Create a pa11y script which can login to the app to capture pages that require authentication [#1044](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1044)
- Update ATO docs and corresponding code docs [#962](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/962)

> - As an OFA admin, I can download raw file [#89](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/89)

Reviewer: I don't have a preference about this, so more of a question for @amilash and @lfrohlich -- are we including epics in this list? as an example, #834 is the ticket that will complete epic #89, and both are in this list 😄

Reviewer 2: I think it would make sense to note something as Epic, and when we complete an Epic have it at the top of the list/call it out.

Reviewer 2: actually, maybe only include the epic when it's closed out? i don't have a strong preference

> - As an OFA admin, I can download raw file [#89](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/89)

Reviewer 2: ```suggestion
- [EPIC] As an OFA admin, I can download raw file [#89](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/89)
```
